docs/app/css/style.css

```diff
@@ -185,10 +185,6 @@ pre {
 pre > code.highlight {
   padding: 16px;
   font-weight: 400;
-  -webkit-user-select: initial;
-  -moz-user-select: initial;
-  -ms-user-select: initial;
```

Comment:
According to MDN, 'initial' is equal to 'none' but it wasn't working. Also I didn't see a point to disabling marking the text on vode samples.